### DISCUSSION TITLE
About identify reaction centers

### DISCUSSION THREAD

**Discussion by thuwilliamwyx** [rdkit/rdkit#6390] [Q&A] [answered]
I have found that someone mentioned rdkit can identify the reaction center (the main atoms that change in a reaction), I am wondering whether rdkit can realize this and which function should I use, thank you!

**Comment by greglandrum** [comment, marked as answer]
@thuwilliamwyx Yes, reactions have a method called `GetReactingAtoms()` which you can use. There's a demonstration of how to use that to highlight reacting atoms and bonds in this blog post: https://greglandrum.github.io/rdkit-blog/posts/2021-11-26-highlighting-changed-bonds-in-reactions.html
